.github/workflows/ci.yaml

```diff
@@ -94,17 +95,6 @@ jobs:
          micromamba list
          cat /proc/cpuinfo || (/usr/sbin/system_profiler SPHardwareDataType; /usr/sbin/sysctl -a | grep machdep.cpu)
 
-    - name: Install pytest and plugins
-      run: |
-         micromamba install pytest pytest-pep8 pytest-cov codecov
-
-    - name: Install GROMACS (${{ matrix.gromacs-version }})
-      # UGLY HACK/FIX (probably to issues with bioconda packages)
-      # - For 3.9, micromamba insists on using pypy 3.9 (and --py-pin does not help).
-      # We can't freeze because then it's not possible to install older GROMACS versions from bioconda. 
```

Comment:
I really hope that this problem doesn't come back in any form because hacking around this dependency hell cost me a lot of sweat and gray hairs.

Reply:
Add a comment above somewhere near where the env.yml file is defined that it is important to install gromacs together with everything else to ensure correct dependency resolution.

Reply:
agreed – i've added some grep checks to confirm that what's been requested has been installed